releasenotes/2.0.0.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: releasenotes
+title: Release Notes for Version 2.0.0
+date: 2016-11-15 12:00:00
+release_notes: true
+permalink: /releasenotes/2.0.0/
+---
+
+1. [Introduction](#intro)
+1. [Release Highlights](#highlights)
+1. [Compute Improvements](#computeimprovements)
+1. [BlobStore Improvements](#blobstoreimprovements)
+1. [Known Issues](#knownissues)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+Apache jclouds 2.0 represents a significant milestone for the project. We are proud to support all major cloud providers in the marketplace with a mature, stable, codebase that is ready for production. You can [read the official announcement here](#). To get started with jclouds, please see the [jclouds installation guide](/start/install/).
+
+## <a id="highlights"></a>Release Highlights
+
+Apache jclouds 2.0 features include:
+
+* Wider compatibility with the Guava and Guice libraries.
```

Comment:
We explicitly support all versions in the compat matrix. We can say that now that we have a build and have done several fixes to support Guava 19.